### DISCUSSION TITLE
Cannot see new (beta) org-wide Project on GitHub mobile

### DISCUSSION THREAD

**Discussion by erikr** [community/community#5520] [Mobile]
Used to be able to assign an issue to a (legacy) Project, but now that I am on the beta for the new Project, I cannot assign an issue to the Project on iOS / iPadOS.

**Comment by aconsuegra** [comment]
Hi @erikr. Thanks for getting in touch! 
You are right. Currently it is not possible to link issues and (new) Projects, but we will be including that functionality soon. Stay tuned!
